lib/matplotlib/backends/qt_compat.py

```diff
@@ -26,8 +27,18 @@
 
 if rcParams['backend'] == 'Qt5Agg':
     QT_RC_MAJOR_VERSION = 5
-else:
+elif rcParams['backend'] == 'Qt4Agg':
     QT_RC_MAJOR_VERSION = 4
+else:
+    # A different backend was specified, but we still got here because a Qt
+    # related file was imported. This is allowed, so lets try and guess
+    # what we should be using.
+    if "PyQt4" in sys.modules:
```

Comment:
For the PyQt4 lines ?
I never used PySide. So, don't know its specificities.

Is the following change correct ?
```python 
if "PyQt4" in sys.modules:
```
=>
```python
if "PyQt4" in sys.modules or "PySide" in sys.modules:
```